bin/yarn

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+set -e
+
```

Comment:
@wfleming question

I was tempted to add `make image` here. The reason: maybe you just cloned the repo and want to add a plugin. If you haven't run `make image`, it will pull the image from dockerhub and then try to run the command. However, we have multiple channels, and it will pull `latest` rather than `eslint-3`, and latest doesn't have yarn yet, and is sort of a different thing.

`make image` gets around that problem, but has the downside that it will need to re-build the image each time the `package.json` or `yarn.lock` change, so it's slow to run it on subsequent yarn operations.

Everything does work nicely as long as you independently run `make image` yourself before you start using `bin/yarn`... so maybe I should just say to do that in `CONTRIBUTING.md`?